src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -67,6 +72,10 @@ export const DepositTx: FC<DepositTxProps> = ({
     : userTokens;
   const sellTokensOptionsMap = keyBy(sellTokensOptions, 'address');
   const selectedSellToken = sellTokensOptionsMap[selectedSellTokenAddress ?? ''];
+  const [tokenAllowance, isLoadingTokenAllowance, getTokenAllowanceErrors] = useAllowance(
```

Comment:
```suggestion
  const [tokenAllowance, isLoadingTokenAllowance, tokenAllowanceError] = useAllowance(
```
Lets remove the `get` since it makes it look like if its a function

Reply:
will do

Reply:
solved on 6ad01db108237880b69ecb49695a74b419be2063